packages/fuels/src/cli/config/forcUtils.ts

```diff
@@ -70,21 +72,21 @@ export function getContractCamelCase(contractPath: string) {
   return camelCase(projectName);
 }
 
-export function getBinaryPath(contractPath: string) {
+export function getBinaryPath(contractPath: string, { buildMode }: FuelsConfig) {
   const projectName = getContractName(contractPath);
-  return join(contractPath, `/out/debug/${projectName}.bin`);
+  return join(contractPath, `/out/${buildMode}/${projectName}.bin`);
 }
 
-export function getABIPath(contractPath: string) {
+export function getABIPath(contractPath: string, { buildMode }: FuelsConfig) {
   const projectName = getContractName(contractPath);
-  return join(contractPath, `/out/debug/${projectName}-abi.json`);
+  return join(contractPath, `/out/${buildMode}/${projectName}-abi.json`);
 }
 
-export function getABIPaths(paths: string[]) {
-  return Promise.all(paths.map((path) => getABIPath(path)));
+export function getABIPaths(paths: string[], config: FuelsConfig) {
+  return Promise.all(paths.map((path) => getABIPath(path, config)));
 }
 
-export const getStorageSlotsPath = (contractPath: string) => {
+export const getStorageSlotsPath = (contractPath: string, { buildMode }: FuelsConfig) => {
   const projectName = getContractName(contractPath);
-  return join(contractPath, `/out/debug/${projectName}-storage_slots.json`);
+  return join(contractPath, `/out/${buildMode}/${projectName}-storage_slots.json`);
```

Comment:
We should probably move `getForcProject` from `test-utils` -> `utils` and add this parameter in so we can utilise it in other places. Ofc it would need the build mode flag.

Reply:
Yes, @nedsalk raised the same point [here](https://github.com/FuelLabs/fuels-ts/pull/1784#discussion_r1497519042).

There are a few differences between the two, but yes, we can consider unifying and normalizing them.